Client/mods/deathmatch/logic/CClientPed.cpp

```diff
@@ -6687,6 +6703,40 @@ bool CClientPed::ExitVehicle()
         return false;
     }
 
+    unsigned char ucDoor = g_pGame->GetCarEnterExit()->ComputeTargetDoorToExit(m_pPlayerPed, pOccupiedVehicle->GetGameVehicle());
```

Comment:
1. Dont use hungarian notation
2. `std::uint8_t` or `auto`

Reply:
Here I didn't change anything, it's original mta code, just moved it above to be accessible

Reply:
Then change the type when you're at it

Reply:
Fixed